src/cmds/init/initialize.ts

```diff
@@ -139,10 +140,10 @@ async function buildWeb3Wallet(argv) {
   let noValidPort = true;
   while (noValidPort) {
     // eslint-disable-next-line require-atomic-updates
-    port = await prompt(`local server port:`, port);
+    const inputPort = (await prompt(`local server port:`, port as unknown as string));
```

Comment:
Did you import types for `prompt` or can we change the type of its second parameter to `string | number`?

Reply:
I can! But it raises issues because prompt takes advantage of the fact that javascript is loosely typed and calls string functions on it such as trim.

Reply:
Wait, better idea:
```suggestion
    const inputPort = (await prompt(`local server port:`, port.toString(10)));
```